pkg/probe/webui_state.go

```diff
@@ -0,0 +1,45 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func probeWebUIState(c http.FortiHTTP, meta *TargetMetadata) ([]prometheus.Metric, bool) {
+	var (
+		mRebootTime = prometheus.NewDesc(
+			"fortigate_last_reboot_seconds",
+			"Last system reboot epoch time in seconds",
+			nil, nil,
+		)
+		mSnapshotTime = prometheus.NewDesc(
+			"fortigate_last_snapshot_seconds",
+			"Last snapshot epoch time in seconds",
+			nil, nil,
+		)
+	)
+
+	type WebUIState struct {
+		SnapshotUTCTime float64 `json:"snapshot_utc_time"`
+		UTCLastReboot   float64 `json:"utc_last_reboot"`
+	}
+
+	type webuiState struct {
+		Results WebUIState
+	}
+
+	var sstate webuiState
```

Comment:
```suggestion
	var state webuiState
```
Maybe use `state` instead of `sstate`, at least for me it is not clear what the first s stands for. ;)